### ISSUE TITLE
ansible-builder will not install EPEL dependencies on anything but CentOS

### ISSUE THREAD

**Issue by felixfontein** [ansible/ansible-builder#542]
So neither on Rocky Linux nor RHEL, even if EPEL is provided.

(On CentOS it will also try to install EPEL dependencies if the EPEL repo hasn't been added.)

It sounds to me like this should be configurable, and potentially investigate whether the EPEL repo is configured instead of simply determining this by looking at the release name.

**Comment by sivel** [comment]
fwiw, this is the reason that line exists: https://github.com/ansible/python-builder-image/commit/6d0d3beac1ced932e82f8838deffd8c7f3426930

> There is no epel repo for ubi8 images, so we need to make this step conditional

One note, is that we previously have said that enabling the epel repo is not something we will do automatically.  So to use it, would need to be a specific action to already have it enabled, such as listing it under system deps.

**Comment by felixfontein** [comment]
Yeah, enabling it automatically is definitely not a good idea, but allowing to configure an option between "always try to install epel dependencies", "never try to install epel dependencies", and somd kind of "autodetect" would be nice (and useful) IMO :)
